### ISSUE TITLE
Scheduled downloads

### ISSUE THREAD

**Issue by technikly** [axcore/tartube#436]
Thanks for taking the time to ask a question! 

### Please read the README
Perhaps your question is answered there

### The authors are not experts in python, security, youtube-dl, FFmpeg, Debian/RPM packaging or Windows programming
You are welcome to ask a Tartube-related question, but you might get a better answer in forums where those kinds of expert can be found

Is there something specific I need to do to get scheduled downloads to start? I've set up a schedule but it never seems to do anything, even considering the 5 minute window. I've tried this as root and as normal user. 

I'm using the latest version of the program on Linux Mint.

I've tried marking things for download, or changing profiles, everything works as intended when I manually click the download button. 

Is there a way to manually start a download via the terminal? I would happily settle for making my own batch.

Thanks,

**Comment by axcore** [comment]
> Is there something specific I need to do to get scheduled downloads to start? I've set up a schedule but it never seems to do anything, even considering the 5 minute window. I've tried this as root and as normal user.

Describe the exact steps you used to create a scheduled download that never starts, for example I just tested this:

- Click Edit > System preferences > Scheduling
- In the `Scheduled download name` box, type `test`
- Click the Add button
- In the new window, click the Start tab
- Set the Interval time to 2 minutes
- Click OK in both windows to close them
- Wait 2 minutes for the scheduled download to start

>Is there a way to manually start a download via the terminal?
No, at the moment Tartube is strictly a GUI.

**Comment by technikly** [comment]
I followed that method exactly, just as you described but nothing happens.
Works all fine when I manually download still. Very confused. Thanks for
the quick reply.

**Comment by axcore** [comment]
Create a **new** scheduled download using that exact method, and tell me what happens.

**Comment by technikly** [comment]
Thanks for the help, but still nothing happened. Nothing showed in the
output, nothing occurred in the progress bar and no files were downloaded.

Is there a way I can send you my config so you can replicate it?

Thanks

**Comment by axcore** [comment]
First run Tartube in a terminal window so we can see any python error messages that are being generated.

Scheduled downloads are stored in the database file (tartube.db inside Tartube's data folder). You can send it to me at lestrol123@gmx.com, but be warned that I will be able to see the videos and channels  you've added to your database.

**Comment by axcore** [comment]
How did you install it?

**Comment by technikly** [comment]
Email sent.

**Comment by axcore** [comment]
The link you emailed me requires signing in to a google account, and I don't have one.

**Comment by technikly** [comment]
Resent! Sorry.

**Comment by axcore** [comment]
Fixed in v2.4.077. 

In the download options window (Edit > General download options...) there's a button marked **Completely reset all download options**. This corrupted the data that handles the **Drag and Drop** tab, so on startup the database never finished loading, so the scheduled downloads didn't start.

As well as fixing the error, I added some failsafe code that shows a message in the **Errors** tab for any similar problem.
